homeassistant/components/yolink/sensor.py

```diff
@@ -58,7 +59,12 @@ class YoLinkSensorEntityDescription(
         if value is not None
         else None,
         exists_fn=lambda device: device.device_type
-        in [ATTR_DEVICE_DOOR_SENSOR, ATTR_DEVICE_TH_SENSOR, ATTR_DEVICE_MOTION_SENSOR],
+        in [
+            ATTR_DEVICE_DOOR_SENSOR,
+            ATTR_DEVICE_TH_SENSOR,
+            ATTR_DEVICE_MOTION_SENSOR,
+            ATTR_DEVICE_VIBRATION_SENSOR,
+        ],
```

Comment:
You have defined a constant for this later on in the file. Why not define the const at the top then use it here?